server/player.lua

```diff
@@ -425,7 +425,7 @@ function CheckPlayerData(source, playerData)
     if source then
         playerData.source = source
         playerData.license = playerData.license or GetPlayerIdentifierByType(source --[[@as string]], 'license2') or GetPlayerIdentifierByType(source --[[@as string]], 'license')
-        playerData.name = GetPlayerName(source)
+        playerData.name = string.sub(GetPlayerName(source):gsub('[^%w%s]', ''), 1, 50)
```

Comment:
The DB allows names to be 255 characters as its set to `varchar(255)`. I don't think the player name should be touched what so ever in my opinion.

Reply:
![image](https://github.com/user-attachments/assets/c7bf3c28-5e5d-4423-bb36-ca4031c25192)
Do I have an old version of the SQL, this was ran from the qbx_core resource, sets the "name" to 50.

Reply:
I do see this now, my apologies not entirely sure why the database set it to 50 for me.

Reply:
nvm I missed this, the player table is altered to set 'name' to `varchar(50)`

https://github.com/Qbox-project/qbx_core/blob/main/qbx_core.sql#L24-L25